kubernetes-manifests/config.yaml

```diff
@@ -39,13 +39,4 @@ data:
   USE_DEFAULT_DATA: "True"
   # All default user accounts are hardcoded to use the login password 'password'
   DEFAULT_LOGIN_PASSWORD: "password"
```

Comment:
Are these `DEFAULT_LOGIN` variables just for the frontend autofill? If so, we should probably rename them to make it clearer, and/or move them to just the frontend service instead of the shared configmap

Reply:
I got a little carried away and renamed everything "Default Data" to "Demo Data".  Let me know if you think that makes the purpose clearer.